scilpy/utils/metrics_tools.py

```diff
@@ -212,3 +211,31 @@ def plot_metrics_stats(mean, std, title=None, xlabel=None,
 
     plt.close(fig)
     return fig
+
+
+def get_roi_metrics_stats(density_map, metrics_files):
+    """
```

Comment:
I don't really mind, but based on François's previous question about trying to have the same nomenclature everywhere:

Everywhere else, it is called _mean_std, not _stats. I think I actually prefer stats but then maybe we should change it everywhere else?

Reply:
Good call, for an other PR.

Reply:
Stats is larger than just mean and std, no? Not sure I would change it to "stats", it is quite large and doesn't precisely represent the content of the method.